homeassistant/components/media_player/kodi.py

```diff
@@ -543,6 +545,25 @@ def media_duration(self):
             total_time['minutes'] * 60 +
             total_time['seconds'])
 
+    @property
+    def media_position(self):
+        """Position of current playing media in seconds."""
+        time = self._properties.get('time')
+
+        if time is None:
+            return None
+
+        return (
+            time['hours'] * 3600 +
+            time['minutes'] * 60 +
+            time['seconds'])
+
+    @property
+    def media_position_updated_at(self):
+        """Last valid time of media position."""
+        if self.state in (STATE_PLAYING, STATE_PAUSED):
+            return dt_util.utcnow()
```

Comment:
This implementation will cause a state change with every update. This method must be implemented such that `media_position_updated_at` doesn't change unless the position has actually changed.

Reply:
I will fix that.

Reply:
I have updated the code and tested it locally now. 

I was inspired by the VLC component.